### PR TITLE
Initial commit to Portfolio

### PR DESCRIPTION
**Pull request by adhyanagpal**
Include some features in portfolio

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,153 @@
...
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1 id="name">Adhya Nagpal</h1>
+      <br>
+      <img src="../public/AdhyaNagpal.png" width="300" height="300">
+      <!-- <p>Click here to get a random message:</p>
+      <button onclick="addRandomGreeting()">Hello</button> -->
```
Comment (monuindia) [inline portfolio/src/main/webapp/index.html:16]: What is the reason to comment these 2 lines? If it was just for testing you can uncomment them. If you no longer need them then it's better to delete those.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,153 @@
...
       <div id="greeting-container"></div>
     </div>
+
+    <div id="intro">
+        <p>Hey there! I'm Adhya. I was born and brought up in Delhi. I'm a coding enthusiast and love to read fiction, play badminton and listen to music in my free time. I've participated in a couple of hackathons like HackIIITD'19 and UHack 4.0 and some open source events like Hacktoberfest'19.</p>
```
Comment (monuindia) [inline portfolio/src/main/webapp/index.html:21]: This line is very large, break it into multiple lines.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (adhyanagpal) [reply portfolio/src/main/webapp/index.html:16]: These 2 lines were there in the original code for testing purposes. I'll remove them. Thanks.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (adhyanagpal) [reply portfolio/src/main/webapp/index.html:21]: Okay, will correct this.

**Pull request merged**